Quaver.Shared/Screens/Edit/UI/Preview/EditorMapPreview.cs

```diff
@@ -48,7 +49,7 @@ public class EditorMapPreview : SelectMapPreviewContainer
         /// <param name="track"></param>
         /// <param name="qua"></param>
         public EditorMapPreview(EditorActionManager manager, Bindable<bool> isPlayTesting, Bindable<SelectContainerPanel> activeLeftPanel,
-            int height, IAudioTrack track = null, Qua qua = null) : base(isPlayTesting, activeLeftPanel, height, track, qua)
+            int height, IAudioTrack track = null, Qua qua = null) : base(isPlayTesting, activeLeftPanel, height, track, qua, true)
```

Comment:
This line is making it unable to compile. Nothing was added to the base class where this is able to be set.